shared/dataflow/codeql/dataflow/internal/DataFlowImpl.qll

```diff
@@ -1394,13 +1380,13 @@ module MakeImpl<LocationSig Location, InputSig<Location> Lang> {
         CcNoCall getCallContextReturn(DataFlowCallable c, DataFlowCall call);
 
         bindingset[cc]
-        LocalCc getLocalCc(Cc cc);
+        LocalCallContext getLocalCc(Cc cc);
```

Comment:
What's going on here? To no longer parameterise over `LocalCc` in the stages seems like a non-trivial change that's unrelated to the PR description. Also, `getLocalCc` is expected to be functional, which now no longer holds.

Reply:
I have reverted this change.